ansible/build-vars.yml

```diff
@@ -0,0 +1,9 @@
+nodejs_app_name: universal
+nodejs_app_install_dir: /opt/{{ nodejs_app_name }}
```

Comment:
Putting `require("universal");` first after `"use strict";` causes the tests to fail with a proper exit code. So something about the require of infusion and kettle coming first, maybe? Bizarre. I'm going to poke a bit more and throw this one to fluid-work.